styles/icons.scss

```diff
@@ -203,6 +203,9 @@
 .icon-warning-large::before {
   content: '\e900';
 }
+.icon-webhooks::before {
+  content: '\e93e';
```

Comment:
<img width="169" alt="image" src="https://user-images.githubusercontent.com/109113616/195698432-9ec5a4d0-6d46-49df-9748-8f4ab8cf8abb.png">

Is it the icon for webhook?

Reply:
it seems like it
https://github.com/determined-ai/determined/pull/5224#issuecomment-1277905953

Reply:
Yes for now; it's the random-hparam search icon until we figure out what else works. I don't know enough about the icon fonts.